### DISCUSSION TITLE
ROCm 6.1 release timeline?

### DISCUSSION THREAD

**Discussion by 2eQTu** [ROCm/ROCm#3017] [Q&A] [answered]
Is the expected release timeline for ROCm 6.1 known?

Obviously this can shift if more bugs are found/fixed, etc.  But basic guidance would be helpful.  I do see folks talking about 6.1RC so I assume it must be soon.  Did I miss the announcement?

Thank you!

**Comment by kentrussell** [comment, marked as answer]
I just pushed the kernel's 6.1 code out about an hour ago, so you should see it coming shortly. Non-automated repos tend to come a little after the manual ones, but you should see it pretty quickly
